### ISSUE TITLE
Add a podcast manually (by URL)

### ISSUE THREAD

**Issue by tofumatt** [mozilla/high-fidelity#68]
This is easy and we should let users add podcasts with a URL if they have one.

**Comment by mcepl** [comment]
Hasn't this been already fixed? Seems to be possible with 885df79988 ?

**Comment by tofumatt** [comment]
Yeah, but we’ll likely be doing a big code update and I want to track individual features as new issues in case we rewrite a fair bit of the code. Using it here to keep track of the new version we’re working on this quarter :-)

- tofumatt
